students/hyska_monika/lesson_14_decorators/exercise1.py

```diff
@@ -0,0 +1,21 @@
+"""
+Write @sort decorator that when applied to a function
+that returns a list, sorts this list,
+"""
+
+
+def sort(func):
+    def func_wrapper():
+        my_list = func()
+        my_list.sort()
```

Comment:
The other way that doesn't modify the list in place is `return sorted(my_list)`. Depending on the situation you should use one or the other.